src/app/components/search/DateRangeFilter.js

```diff
@@ -305,9 +306,10 @@ const DateRangeFilter = ({
         getStartDateStringOrNull(),
         getEndDateStringOrNull(),
       ));
-    } else if (rangeType === rangeTypes.relative) {
+    } else if (['less_than', 'more_than'].indexOf(rangeType) !== -1) {
```

Comment:
I suggest changing this comparison to `['less_than', 'more_than'].includes(rangeType)` for better readability

Reply:
Done